src/components/MdComponents/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { ComponentProps, type HTMLAttributes } from "react"
+import { Badge, Box, type BoxProps } from "@chakra-ui/react"
+
+import type { ChildOnlyProp } from "@/lib/types"
+
+import ButtonDropdown, {
+  type ButtonDropdownProps,
+} from "@/components/ButtonDropdown"
+import Contributors from "@/components/Contributors"
+import MarkdownImage from "@/components/MarkdownImage"
+import { mdxTableComponents } from "@/components/Table"
+import TooltipLink from "@/components/TooltipLink"
+import YouTube from "@/components/YouTube"
+
+import { cn } from "@/lib/utils/cn"
+
+import ContributorsQuizBanner from "../Banners/ContributorsQuizBanner"
+import Card from "../Card"
+import DocLink from "../DocLink"
+import Emoji from "../Emoji"
+import ExpandableCard from "../ExpandableCard"
+import FeaturedText from "../FeaturedText"
+import GlossaryTooltip from "../Glossary/GlossaryTooltip"
+import IdAnchor from "../IdAnchor"
+import InfoBanner from "../InfoBanner"
+import IssuesList from "../IssuesList"
+import LocaleDateTime from "../LocaleDateTime"
+import MainArticle from "../MainArticle"
+import { StandaloneQuizWidget } from "../Quiz/QuizWidget"
+import { ButtonLink } from "../ui/buttons/Button"
+import { Flex } from "../ui/flex"
+import { ListItem, OrderedList, UnorderedList } from "../ui/list"
+
+export const commonHeadingAttributes = (className: string, id?: string) => ({
+  id,
+  className: cn(
+    "font-bold leading-xs my-8",
```

Comment:
We will need this as important to avoid being overridden by the global heading sizes
```suggestion
    "font-bold !leading-xs my-8",
```

Reply:
@pettinarip doing this will override the leading for smaller screen sizes, such as what is used in the h2 for the Docs layout [as discussed above](https://github.com/ethereum/ethereum-org-website/pull/13620#discussion_r1745251335)

I'd remove that leading in favor of establishing this override, as I cannot recognize a big difference.